juniper_official/chassis/check-routing-engine-temperature.rule

```diff
@@ -55,7 +55,7 @@ healthbot {
             }
             field routing-engine-temperature {
                 sensor components-oc {
-                    where "/components/component/properties/property/@name == 'RE temperature'";
+                    where "/components/component/properties/property/@name == 'temperature'";
```

Comment:
This change seems to be contradicting https://github.com/Juniper/healthbot-rules/pull/677